docs/index.html

```diff
@@ -524,6 +526,8 @@ <h1 class="display-4 theme-content-element-title">Button</h1>
 	</span>
 	<div class="theme-content-element-description"><p>The buttons are to be used across the various internal systems. If there is only one button on the current screen, this should always be a primary button. When you use multiple buttons, the primary button appears on the right and any secondary buttons appear on the left. Secondary button should not be used alone on a screen. Action buttons are specially designed for the user to take an action that will affect something else.</p>
 <p>Changing the color of a button to convey purpose is not ideal as people with vision impairments may struggle. Usage of other colors should be properly discussed and the context clear.</p>
+<h3>✅ When to use:</h3><p>The buttons should have a min left and right padding between text and border of 16 px (2 standard spacing). The minimum width of a button must be 90 px. The icon should be placed to the left of the text and the distance between text and icon should be 1 (default distance).</p>
```

Comment:
er ikke det noe vi skal gjøre og ikke de som bruker hdd?

Reply:
Är anbefallt, men om det finns flera kort på rad/höjd som innehåller olika mycket information så borde korten hellre vara lika stora/passa in i gridet än att de har exakt den paddingen. Vad tänker dere?

Reply:
Denne teksten i HDD er ment for de som skal bruke HDD, og ikke oss. Padding og border er satt av oss for de så de trenger ikke å vite om det

Reply:
Kan la dette stå kanskje? "The minimum width of a button must be 90 px."

Reply:
Er noe vi setter på button classen så ingen vil kunne få mindre knapper ved annet de hacker på css'en, og da er de ut å kjører uansett. Og min size bør ikke være i PX, da dette ikke tar høyde for endring av font size

Reply:
Ja det er logisk. Det er noe som må legges til på button!